internal/v2/tls_config_store/tls_config_store.go

```diff
@@ -0,0 +1,114 @@
+// Static implementation of TLS Configuration Store (no calls to S2Av2, Remote Signer Library, Certificate Verifier)
+package tls_config_store
+
+import (
+	"log"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	_ "embed"
+)
+
+var (
+	//go:embed example_cert_key/client_cert.pem
+	clientCert []byte
+	//go:embed example_cert_key/server_cert.pem
+	serverCert []byte
+	//go:embed example_cert_key/client_key.pem
+	clientKey []byte
+	//go:embed example_cert_key/server_key.pem
+	serverKey []byte
+)
+
+func VerifyPeerCertificateFunc(instanceName string, pool *x509.CertPool) func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
```

Comment:
nit: I think the VerifyPeerCertificateFunc should be unexported (i.e. "verify" should be lowercase) because we don't want to use it outside of this package.

Also, I think unexported functions should be moved to the bottom of the file (i.e. list the public ones first).

Reply:
Great point, thanks!